build.gradle

```diff
@@ -41,7 +41,7 @@ allprojects {
   apply plugin: 'groovy'
 
   ext {
-    spinnakerDependenciesVersion = project.hasProperty('spinnakerDependenciesVersion') ? project.property('spinnakerDependenciesVersion') : '1.0.24'
+    spinnakerDependenciesVersion = project.hasProperty('spinnakerDependenciesVersion') ? project.property('spinnakerDependenciesVersion') : '1.1.0-SNAPSHOT'
```

Comment:
snapshot eh?

Reply:
Yes, from the PR description: `Depends on spinnaker/spinnaker-dependencies#243`